examples/notebooks/introduction_enlyze_python_sdk.ipynb

```diff
@@ -0,0 +1,2981 @@
+       "2023-11-21 22:00:00+00:00                 -6.400000   \n",
+       "2023-11-21 22:10:00+00:00                 -6.483333   \n",
+       "2023-11-21 22:20:00+00:00                 -6.450000   \n",
+       "\n",
+       "                           Extruder A Schneckenumdrehung  \\\n",
+       "time                                                       \n",
+       "2023-11-21 21:40:00+00:00                            0.0   \n",
+       "2023-11-21 21:50:00+00:00                            0.0   \n",
+       "2023-11-21 22:00:00+00:00                            0.0   \n",
+       "2023-11-21 22:10:00+00:00                            0.0   \n",
+       "2023-11-21 22:20:00+00:00                            0.0   \n",
+       "\n",
+       "                           Extruder B Durchsatz  Extruder A Förderrate  \\\n",
+       "time                                                                     \n",
+       "2023-11-21 21:40:00+00:00                   0.0                    0.0   \n",
+       "2023-11-21 21:50:00+00:00                   0.0                    0.0   \n",
+       "2023-11-21 22:00:00+00:00                   0.0                    0.0   \n",
+       "2023-11-21 22:10:00+00:00                   0.0                    0.0   \n",
+       "2023-11-21 22:20:00+00:00                   0.0                    0.0   \n",
+       "\n",
+       "                           Extruder B Schmelzetemperatur  \n",
+       "time                                                      \n",
+       "2023-11-21 21:40:00+00:00                           20.0  \n",
+       "2023-11-21 21:50:00+00:00                           20.0  \n",
+       "2023-11-21 22:00:00+00:00                           20.0  \n",
+       "2023-11-21 22:10:00+00:00                           20.0  \n",
+       "2023-11-21 22:20:00+00:00                           20.0  "
+      ]
+     },
+     "execution_count": 34,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data_resampled.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "609024e5-d407-49f8-9f28-0aa81463ca7f",
+   "metadata": {},
+   "source": [
+    "### Data preparation and exploration\n",
+    "\n",
+    "Data preparation and exploration is a bigger topic that will be covered in later tutorials, but here are a few examples of how to get started. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "id": "d05a9b60-e954-416e-9258-d96084281d6f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# quick step to sort the columns alphabetically\n",
+    "data_resampled = data_resampled[sorted(data_resampled.columns)]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "id": "8f7fc1a4-b41e-48b9-b402-18d429f2c550",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# reset index and convert time to local timezone\n",
+    "data_resampled = data_resampled.reset_index()\n",
```

Comment:
I'd like to understand why resetting the index is done so often. I expected that the timestamp would actually be the best index we can have by default.